doc/source/releases/24.2_announce.rst

```diff
@@ -1,12 +1,314 @@
 
-:orphan:
+===========================================================
+24.2 Galaxy Release (February 2025)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+Feature1
+--------
+
+Feature description.
+
+Feature2
+--------
+
+Feature description.
+
+Feature3
+--------
+
+Feature description.
+
+DRAFT CONTENT
+
+* Add wizard-like export view for workflow invocations
+* Workflow Editor Activity Bar + Undo stack UI
+* Masthead Revision
+* Guide users to collection builders
+* Workflow Invocation view improvements
+* Libraries Modernisation: Directory Dataset Picker
+* Improvements to help terms & tool help.
+* Feature - stdout live reporting
+* Allow OAuth 2.0 user defined file sources (w/Dropbox integration)
+
+Please see the `24.2 user release notes <24.2_announce_user.html>`__ for a summary of new user features.
+
+Get Galaxy
+===========================================================
+
+The code lives at `GitHub <https://github.com/galaxyproject/galaxy>`__ and you should have `Git <https://git-scm.com/>`__ to obtain it.
+
+To get a new Galaxy repository run:
+  .. code-block:: shell
+
+      $ git clone -b release_24.2 https://github.com/galaxyproject/galaxy.git
+
+To update an existing Galaxy repository run:
+  .. code-block:: shell
+
+      $ git fetch origin && git checkout release_24.2 && git pull --ff-only origin release_24.2
+
+See the `community hub <https://galaxyproject.org/develop/source-code/>`__ for additional details on source code locations.
+
+
+Admin Notes
+===========================================================
+
+DRAFT CONTENT
+
+* Add job metrics per invocation
+* Allow OAuth 2.0 user defined file sources (w/Dropbox integration)
+* Support high-availability setups for the interactive tools proxy
+* Backend handling of setting user-role, user-group, and group-role associations
+* Decouple user email from role name
+* Add unique constraints to the email and username fields in the galaxy_user table
```

Comment:
This isn't of concern to admins directly, however it should be mentioned that exclusive locks are required for the migration, and so it is recommended to shut down all galaxy processes.